tools/docker/Dockerfile

```diff
@@ -264,11 +266,23 @@ SHELL ["/bin/bash", "-c"]
 # Test image that starts up containerd and the naive snapshotter. The default CMD will drop to a bash shell. Overrides
 # to CMD will be provided appended to /bin/bash -c
 FROM firecracker-containerd-unittest as firecracker-containerd-e2etest-naive
+# Install tini init to handle zombies created by our double-forked shims.
```

Comment:
I've now encountered the issue with needing an init to reap zombies outside this PR, so I've updated #170 with this change to the Dockerfile and will be removing it from this PR